.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_size = 2
```

Comment:
That's a fair point; only place in README I can think of is a code indent and I can't think of a situation where we'd use one. It won't throw any errors/try to convert existing indentations will it? Thinking if someone accidentally opens the LICENSE file